libs/coin-algorand/src/types.ts

```diff
@@ -55,11 +57,20 @@ export type AlgorandOperationExtra = {
   memo?: string;
   assetId?: string;
 };
+export function isAlgorandOperationExtra(op: OperationExtra): op is AlgorandOperationExtra {
```

Comment:
This is to [narrow](https://www.typescriptlang.org/docs/handbook/2/narrowing.html) the type during OperationExtra serialization.

Reply:
Either we narrow or cast the type... but since we will never use coin-algorand to implement a polkadot app IMO casting should be enough. It's minor anyway

Reply:
"Force casting" is not a good practice. Those "narrow" functions are not to complex to implement

Reply:
If we are 100% sure there will never be something else in it I think this is fine. Anyway long term we will have Algorand specific types so i'm fine with it